### ISSUE TITLE
Styling on reset kubernetes confirmation dialog does not match the rest of RD

### ISSUE THREAD

**Issue by adamkpickering** [rancher-sandbox/rancher-desktop#2636]
### Actual Behavior

Styling on dialogs do not match styling on other dialogs or the rest of RD.

### Steps to Reproduce

For kubernetes confirmation dialog: Select a lower version of kubernetes, and then click Apply. Dialog should pop up.

For cancel apply dialog: select a lower version of kubernetes, then click Cancel. Dialog should pop up.

### Result

Styling on dialogs do not match styling on other dialogs or the rest of RD.

### Expected Behavior

Styling should match.

### Additional Information

![image](https://user-images.githubusercontent.com/19395776/181125116-46c92e1e-a900-40d8-b650-31485c871228.png)
![image](https://user-images.githubusercontent.com/19395776/181126253-865f9b9f-5351-4370-8e51-ce43605d06a8.png)

### Rancher Desktop Version

171c29e1

### Rancher Desktop K8s Version

1.24.3

### Which container runtime are you using?

containerd (nerdctl)

### What operating system are you using?

Ubuntu

### Operating System / Build Version

Ubuntu 22.04

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

deb

**Comment by mook-as** [comment]
FWIW, those dialogs are Gnome-native, hence why they don't match RD. On Windows, we get Windows 7 style ones instead:

!["Close preferences without applying" dialog on Windows](https://user-images.githubusercontent.com/3977982/181126685-df8c146c-b6b8-44cb-ad16-01413f27ec07.png)

**Comment by adamkpickering** [comment]
I see the same thing with the confirmation dialog when bulk deleting images:
![image](https://user-images.githubusercontent.com/19395776/181298227-9ef16ff4-f6be-47a9-b0cf-472f8c5358c8.png)

**Comment by rak-phillip** [comment]
Yeah, those are native dialogs raised through electron. The style of these will vary based on OS.

We started work on a custom dialog component with the goal that it should consume the electron dialog API. I'd like to settle on a consistent style throughout, but more work is needed in this area before it can be implemented everywhere. 

https://github.com/rancher-sandbox/rancher-desktop/blob/171c29e1ddd4865fa026a237bb22dd7f219aba57/src/pages/Dialog.vue
